lib/ddtrace.rb

```diff
@@ -11,23 +13,48 @@ module Datadog
   #   span = Datadog.tracer.trace('web.request')
   #   span.finish()
   #
+  # If you want to override the default tracer, the recommended way
+  # is to "pin" your own tracer onto your traced component:
+  #
+  #   tracer = Datadog::Tracer.new
+  #   pin = Datadog::Pin.get_from(mypatchcomponent)
+  #   pin.tracer = tracer
+
   def self.tracer
     @tracer
   end
 end
 
+begin
+  # We include 'redis-rails' here if it's available, doing it later
+  # (typically in initialize callback) does not work, it does not
+  # get loaded in the right context.
+  require 'redis-rails'
+  Datadog::Tracer.log.info("'redis-rails' module found, datadog redis integration is available")
+rescue LoadError
+  Datadog::Tracer.log.info("no 'redis-rails' module found, datadog redis integration is not available")
+end
+
+# Autopatching non-Rails contribs if env var is set
+Datadog::Monkey.autopatch_all
```

Comment:
not totally sure that this is what we want... is the Python client using this approach of using an env var to ``autopatch_all``?